static/documents/tradetrust/v2/ebl-goerli.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": "https://schema.openattestation.com/2.0/schema.json",
+  "data": {
+    "name": "02736f77-d18a-4ce4-867a-5e680337aedd:string:TradeTrust Bill of Lading v2",
+    "shipper": {
+      "address": {
+        "street": "31f5f70e-8d01-42d0-afad-45e7bfc5f959:string:One North",
+        "country": "4e8c79ab-922d-44cc-8d81-7a3c4f7faf9a:string:Singapore"
+      },
+      "name": "199a9550-2a6d-4edd-a8f0-b067ed290ac4:string:Demo Shipper"
+    },
+    "consignee": {
+      "name": "b963cd9b-1273-494a-a5d5-2385255b48a0:string:Demo Consignee"
+    },
+    "notifyParty": {
+      "name": "5caae3b7-9e3c-4f0c-b65b-780594344c1f:string:Demo Notify"
+    },
+    "$template": {
+      "type": "c0644a87-03dd-4ba3-a6bd-8005eab66569:string:EMBEDDED_RENDERER",
+      "name": "9ebf8a4d-4664-40e1-9749-e7d0270170f8:string:BILL_OF_LADING",
+      "url": "61e2d3d8-8ec0-402f-84a2-7acbf78b27d1:string:https://generic-templates.tradetrust.io"
+    },
+    "issuers": [
+      {
+        "identityProof": {
+          "type": "24917ac0-3a24-4dce-a80d-49283cc7ba6c:string:DNS-TXT",
+          "location": "9bda2a89-9cf1-4507-8b5a-b0a5fd4a0f57:string:demo-tradetrust.openattestation.com"
+        },
+        "name": "ca252a45-d6ba-420c-96bc-8e09cdd4c2c2:string:Demo token registry",
+        "tokenRegistry": "43b6251d-fd09-428b-8f87-9d1c8a3ace29:string:0x921dC7cEF00155ac3A33f04DA7395324d7809757"
```

Comment:
Just to check, this is the v2 token registry because only this version will work with the current tests is that right?

Reply:
um that i did not try. there is not much tests here so probably didn't matter. no particular reason for tk v2, i didn't use v4 yet cos haven merged